src/types.js

```diff
@@ -167,6 +177,8 @@ export type JsonCityType = {
   live: boolean,
   events: boolean,
   extras: boolean,
+  tunews: boolean,
+  ["push-notifications"]: boolean,
```

Comment:
Phew, that seems pretty inconsistent. We should use an underscore here (just like with `name_without_prefix`).
@klinzo @maxammann Do you agree ?

Reply:
I agree.
@svenseeberg can we change this?

If yes then I think it should be easy to fix in this PR.